chapter11/ajd/exercise2.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use IslandDate;
+
+my ( $mday, $mon, $year, $wday ) = ( localtime )[ 3 .. 6 ];
+
+$wday = IslandDate::number_to_day_name( $wday );
+$mon = IslandDate::number_to_month_name( $mon );
+$year += 1900;
```

Comment:
had you used Time::Piece earlier, you can still get at 0 based months and days of the week, and you would also not need to do the crazy `+= 1900`